includes/classes/RegisteredDataHandler.php

```diff
@@ -0,0 +1,640 @@
+<?php
+/**
+ * Class for process registered data.
+ *
+ * @package  distributor
+ */
+
+namespace Distributor;
+
+/**
+ * This class is responsible for processing the registered data for the post content and post meta.
+ *
+ * @since x.x.x
+ */
+class RegisteredDataHandler {
+	/**
+	 * The Connection data array.
+	 *
+	 * @since x.x.x
+	 * @var array
+	 */
+	public $connection_data = array();
+
+	/**
+	 * Constructor for the RegisteredDataHandler class.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param array $connection_data The connection data array.
+	 */
+	public function __construct( $connection_data = array() ) {
+		$this->connection_data = $connection_data;
+	}
+
```

Comment:
Extra space here

Reply:
Resolved in https://github.com/10up/distributor/pull/1309/commits/d7be301a3e53ac3c9c0340e99e9b6359a67f7487